### PR TITLE
CodeSandbox CI configuration

### PR DESCRIPTION
**Pull request by baseten**
This PR adds configuration for CodeSandbox CI along with a basic local sandbox template which will be forked when testing PRs.

Because the monorepo packages don't contain their own version numbers an additional script was also necessary to write these so that CodeSandbox CI can publish to its registry.

### CODE REVIEW

File: .codesandbox/ci.json
Comment (Zyie) [inline .codesandbox/ci.json:5]: we are missing `pixi.js-legacy` here

File: .codesandbox/ci.json
Comment (baseten) [reply .codesandbox/ci.json]: Nice, I actually just pushed this change as well 😄 

I was initially trying to avoid spamming the CodeSandbox CI registry with too many modules, but I don't think it will make a difference

**Comment by baseten** [comment]
Annoyingly globs don't seem to work, the CSB registry doesn't have a working pixi version: https://codesandbox.io/s/pixi-js-sandbox-63pdvm

**Comment by SuperSodaSea** [comment]
Version at commit 3a7bf64454f64a9f93466c8d4e3464375063b829 (just publish everything) is working: <https://codesandbox.io/s/pixi-js-sandbox-jurgtz>
I think this is probably a random bug of CodeSandbox.

**Comment by SuperSodaSea** [comment]
It looks like others have encountered this problem:
- <https://github.com/codesandbox/codesandbox-client/issues/3748>
- <https://github.com/codesandbox/codesandbox-client/issues/6825>

**Comment by SuperSodaSea** [comment]
After triggering rebuild x2 it is working again: <https://codesandbox.io/s/pixi-js-sandbox-sco4xi>

**Review by bigtimebuddy — APPROVED** [review]
Nice work @baseten

File: package.json
Comment (bigtimebuddy) [inline package.json:35]: Would this be more efficient if we remove the doc building, linting and type checking? Is that necessary?

```suggestion
    "codesandbox-ci": "run-s build build:types prepublish-ci",
```

**Comment by SuperSodaSea** [comment]
↑ Should we revert this commit since we have #9282?

**Comment by baseten** [comment]
> 

Yes, it's in pr

**Comment by baseten** [comment]
> ↑ Should we revert this commit since we have #9282?

Yes, in progress already 👍

**Comment by baseten** [comment]
@bigtimebuddy I've made your suggested script change 👍 

If possible, assuming you're happy with https://github.com/pixijs/pixijs/pull/9282 can we hold off merging this until that is merged and then we should be able to see the whole workflow run once this PR is merged 😄

**Pull request merged**
